README.md

```diff
@@ -17,23 +17,24 @@ We are explicitly soliciting contributors to maintain those integrations labelle
+| **OpenVPN**       | [Github: oqs-demos/openvpn](openvpn)                     | [Dockerhub: openquantumsafe/openvpn](https://hub.docker.com/repository/docker/openquantumsafe/openvpn)                                                                                                                                      | unsupported
+| **ngtcp2**        | [Github: oqs-demos/ngtcp2](ngtcp2)                       | Dockerhub: [Server: openquantumsafe/ngtcp2-server](https://hub.docker.com/repository/docker/openquantumsafe/ngtcp2-server), [Client: openquantumsafe/ngtcp2-client](https://hub.docker.com/repository/docker/openquantumsafe/ngtcp2-client) | unsupported
+| **OpenLiteSpeed** | [Github: oqs-demos/openlitespeed](openlitespeed)         | [ Dockerhub: openquantumsafe/openlitespeed](https://hub.docker.com/repository/docker/openquantumsafe/openlitespeed)                                                                                                                         | unsupported
+| **h2load**        | [Github: oqs-demos/h2load](h2load)                       | [ Dockerhub: openquantumsafe/h2load](https://hub.docker.com/repository/docker/openquantumsafe/h2load)                                                                                                                                       | unsupported
+| **HAproxy**       | [Github: oqs-demos/haproxy](haproxy)                     | [Dockerhub: openquantumsafe/haproxy](https://hub.docker.com/repository/docker/openquantumsafe/haproxy)                                                                                                                                      | unsupported
+| **Mosquitto**     | [Github: oqs-demos/mosquitto](mosquitto)                 | [Dockerhub: openquantumsafe/mosquitto](https://hub.docker.com/repository/docker/openquantumsafe/mosquitto)                                                                                                                                  | unsupported
+| **Envoy**         | [Github: oqs-demos/envoy](envoy)                         | [ Dockerhub: openquantumsafe/envoy](https://hub.docker.com/repository/docker/openquantumsafe/envoy)                                                                                                                                         | unsupported
+| **Unbound**       | [Github: oqs-demos/unbound](unbound)                     | [ Dockerhub: openquantumsafe/unbound](https://hub.docker.com/repository/docker/openquantumsafe/unbound)                                                                                                                                     | unsupported
+| **Locust**        | [Github: oqs-demos/locust](locust)                       | [ Dockerhub: openquantumsafe/locust](https://hub.docker.com/repository/docker/openquantumsafe/locust)                                                                                                                                       | @davidgca
```

Comment:
Given that this demo is currently working and supported, I think it can move closer to the top of the list :)

Reply:
Fix in this [commit](https://github.com/open-quantum-safe/oqs-demos/pull/304/commits/91769ca63c25a4b9fb393634cf6ae2397f87d132)